target/linux/rtl838x/profiles/00-default.mk

```diff
@@ -0,0 +1,13 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+define Profile/Default
+	NAME:=Default Profile
+	PRIORITY:=1
+endef
+
+define Profile/Default/Description
+	Default package set compatible with most boards.
+endef
```

Comment:
Though other files do it differently, I'd add an empty line here before the eval.

Reply:
Done

Reply:
Don't see the empty line, but that's really the least important thing.